macros/ui/niceTables.pl

```diff
@@ -405,7 +361,44 @@ =head2 Options for ROWS
 
 =head2 Options for COLUMNS
 
-Column styling is handled indirectly for now, mostly through the texalignment option above.
+Column styling is handled indirectly for now, mostly through the C<texalignment> option above.
+
+=head2 Deprecations
+
```

Comment:
I think it may be helpful to include the alternative here.  I'm guessing instead of `tablecss => 'font-family: fantasy; text-decoration: underline;'>` to use
```
tablecss => { fontFamily => 'fantasy', textDecoration => 'underline' }
```

Reply:
Up at line 161 describes the new way to to declare CSS. Did you see that there?

Reply:
I did see that and know that you can put them in either way.  I read this as the camelCase version was the preferred method and if that is the case, we should use examples in the preferred way.  If it doesn't matter, let's leave it.

Reply:
Either `fontFamily` or `'font-family'` is OK to use. If I had to make a choice, I would support `'font-family'` because that is the actual CSS key. The other is just a convenience that is consistent with the js convention for referencing a CSS key. But yes, both are meant to be supported. And both are new at the same time, not being deprecated.